hax/hax/handler.py

```diff
@@ -217,6 +217,17 @@ def update_process_failure(self, planner: WorkPlanner,
                                                   chp_type=proc_type,
                                                   chp_pid=0,
                                                   fid=state.fid))
+
+                                if current_status == ObjHealth.OFFLINE:
```

Comment:
Yes, makes sense. Will indeed make the code more readable. 
the purpose I was doing here was to avoid the additional checks of the process being under the local or RC node. 

Will make the changes and retest it.

Reply:
Yes DONE